ncclient/manager.py

```diff
@@ -58,7 +58,7 @@
 """
 
 
-def make_device_handler(device_params):
+def make_device_handler(device_params, ignore_errors):
```

Comment:
This is not a backwards compatible change. `make_device_handler` is not a private API and we cannot change the signature this way.

Reply:
Fixed by setting default None value